pkg/storaged/dialogx.jsx

```diff
@@ -280,7 +280,7 @@ export const PassInput = (tag, title, options) => {
         initial_value: "",
 
         render: (val, change) =>
-            <input data-field={tag}
+            <input data-field={tag} data-field-type="text-input"
```

Comment:
As mentioned in other PRs, the `data-*` attributes are not preferred in React. They exist just for compatibility with (older) libraries.
Wouldn't it be better to determine the type form css `class`? If not in all cases recently, we can add a new one, without it's definition.

Reply:
We might use a class for `data-field-type`, but it's more difficult for `data-tag`.  But I don't see why we should stop using `data-*` attributes.  They work, React doesn't warn about them, and there isn't a equivalent alternative, right?